app/controllers/dividends/DividendsCYAController.scala

```diff
@@ -112,4 +119,11 @@ class DividendsCYAController @Inject()(
     }
   }
 
+  private def auditSubmission(details: CreateOrAmendDividendsAuditDetail)
+                                        (implicit hc: HeaderCarrier,
+                                         executionContext: ExecutionContext): Future[AuditResult] = {
+    val event = AuditModel("createOrAmendDividendUpdate", "createOrAmendDividendUpdate", details)
```

Comment:
Could you just add an 's' onto these so the spelling matches everything else?

```suggestion
    val event = AuditModel("createOrAmendDividendsUpdate", "createOrAmendDividendsUpdate", details)
```